pkg/module/resources.go

```diff
@@ -111,10 +90,10 @@ func generateResources(log *zap.SugaredLogger, version string, credLabel []byte,
 			r.Input.Type = "file"
 		}
 
-		if len(credLabel) != 0 {
+		if len(credMatchLabels) != 0 {
```

Comment:
I don't know why, but it looks strange to me, I'd prefer the following:
```suggestion
		if len(credMatchLabels) > 0 {
```